ts/haskell/initializers.ts

```diff
@@ -29,14 +29,16 @@ export const mkHaskellProjectInitializer: Initializer = (dir) => {
   }
   const parsedCabal = JSON.parse(decoder.decode(jsonParseResult.stdout));
 
+  const executables = parsedCabal.executables
+    ? `\n  executables: ${JSON.stringify(parsedCabal.executables)},`
+    : "";
```

Comment:
In the tests you assert an empty array. Do we want to make this optional? Maybe it makes sense to always print this even if it's an empty array in the initializer?

Reply:
Ah right, sorry I was confusing undefined with empty array. Does it make sense to template this out even if `parsedCabal.executables` is undefined?